custom_test.py

```
@@ -0,0 +1,130 @@
+from datetime import datetime, timezone
+from typing import Optional, Any
+
+from mad_spark_alt.core import SmartQADIOrchestrator
+from test_utils import truncate_text, validate_qadi_result
+
+
+async def test_with_custom_prompt(
+    problem_statement: str,
+    context: str = "",
+    max_ideas: int = 2,
+) -> Optional[Any]:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `test_with_custom_prompt`

(ANN401)